File: trabajo 6/trabajo 6.js
const matriz = [
    [1, 2, 3],
    [4, 5, 6],
    [7, 8, 9]
];

console.log("Recorrido por columnas:");
for (let j = 0; j < 3; j++) {         // Columnas
    for (let i = 0; i < 3; i++) {     // Filas
        process.stdout.write(matriz[i][j] + " ");
    }
}
console.log();

File: trabajo 6/trabajo 6.py
matriz = [
    [1, 2, 3],
    [4, 5, 6],
    [7, 8, 9]
]

print("Recorrido por columnas:")
for j in range(len(matriz[0])):
    for i in range(len(matriz)):
        print(matriz[i][j], end=" ")